libcxx/include/__exception/exception_ptr.h

```diff
@@ -26,6 +28,22 @@ namespace std { // purposefully not using versioning namespace
 class _LIBCPP_EXPORTED_FROM_ABI exception_ptr {
   void* __ptr_;
 
+#  ifndef _LIBCPP_HAS_NO_EXCEPTIONS
+#    if defined(LIBCXXRT) || defined(LIBCXX_BUILDING_LIBCXXABI)
```

Comment:
Why does this need to be conditional on libcxxrt or libcxxabi, if libstdc++/libsupc++ also implements this ABI?

Reply:
Initially i didn't want to do that due to some amount of copy-paste, but you're right, why not